game-engine/src/main/scala/za/co/entelect/challenge/game/contracts/commands/UseOilCommand.scala

```diff
@@ -5,21 +5,21 @@ import za.co.entelect.challenge.game.contracts.map.{BlockPosition, CarGameMap}
 
 class UseOilCommand extends BaseCarGameCommand {
 
-  override def getFuturePositionAfterAdditionalProcessingOfCommand(carGameMap: CarGameMap,
-                                                                   carGamePlayer: CarGamePlayer,
-                                                                   currentPlayerPosition: BlockPosition):
-  BlockPosition = {
-    val futurePosition: BlockPosition = CommandHelper.getFuturePosition(carGamePlayer, currentPlayerPosition)
-    val blockToPlaceOil: BlockPosition = CommandHelper.getBlockToPlacePowerUp(carGamePlayer, currentPlayerPosition)
+    override def getFuturePositionAfterAdditionalProcessingOfCommand(carGameMap: CarGameMap,
+                                                                     carGamePlayer: CarGamePlayer,
+                                                                     currentPlayerPosition: BlockPosition):
+    BlockPosition = {
+        val futurePosition: BlockPosition = CommandHelper.getFuturePosition(carGamePlayer, currentPlayerPosition)
+        val blockToPlaceOil: BlockPosition = currentPlayerPosition;
```

Comment:
Why add a semicolon `;` for a language that don't use them? 😋

Reply:
Will remove them 👍 , all of them (^__^)